app/Services/File.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace App\Services;
+
+class File
+{
+    /**
+     * Generate a file with content
+     * 
+     * @param string output - the path to the new file
+     * @param string result - the content of the new file
+     */
+    public function createFile( $output, $result )
+    {
+        // Create the directory the file is supposed to be located in 
+        if( strpos($output, '/') ){
+
+            // Everything before the last none item in the array is the path to the file
```

Comment:
Unless I'm missing something, this can be replaced with https://www.php.net/manual/en/function.dirname.php ?

Reply:
See, you've just came back to coding with PHP after decades, and you're still an expert! right on! will use this